flink-core/src/main/java/org/apache/flink/api/common/ArchivedExecutionConfig.java

```diff
@@ -54,6 +54,19 @@ public ArchivedExecutionConfig(ExecutionConfig ec) {
 		}
 	}
 
+	public ArchivedExecutionConfig(
```

Comment:
What do you mean? The other constructor is only used once if IntelliJ is to be trusted in `ExecutionConfig.archive`. But I'm fine with leaving it as is.